app/scripts/modules/core/region/regionSelectField.directive.html

```diff
@@ -7,7 +7,7 @@
             ng-model="component[field]"
             ng-change="onChange()"
             required>
-      <option ng-repeat="region in regions" value="{{region.name}}" ng-selected="component[field] === region.name">{{region.name}}</option>
+      <option ng-repeat="region in regions" value="{{region.name}}" ng-selected="component[field] === region.name">{{region.name}} {{region.deprecated ? '(deprecated)' : ''}}</option>
```

Comment:
Do we have the account name available here? Might be nice to include it, i.e. `(deprecated in test)`

No big deal if not.